po/r-novice-gapminder.ja.po

```diff
@@ -597,10 +597,11 @@ msgid ""
 "[OSI][osi]-approved\n"
 "[MIT license][mit-license]."
 msgstr ""
-"Except where otherwise noted, the example programs and other software\n"
-"provided by Software Carpentry and Data Carpentry are made available under the\n"
-"[OSI][osi]-approved\n"
-"[MIT license][mit-license]."
+"Except where otherwise noted, the example プログラム and other ソフトウェア\n"
+"provided by <ruby>Software Carpentry<rp>【</rp><rt>ソフトウェアカーペントリー</rt><rp>】</rp></ruby>と\n"
+"<ruby>Data Carpentry<rp>【</rp><rt>データーカーペントリー</rt><rp>】</rp></ruby>は\n"
+"[OSI][osi]の検定済みの\n"
+"[MIT license](https://ja.osdn.net/projects/opensource/wiki/licenses%2FMIT_license)の下で利用可能です。"
```

Comment:
```suggestion
"[MIT ライセンス](https://ja.osdn.net/projects/opensource/wiki/licenses%2FMIT_license)の下で利用可能です。"
```
A link may need to be replaced with: https://licenses.opensource.jp/MIT/MIT.html